src/component/form/form.js

```diff
@@ -14,7 +14,9 @@ class Form extends Component {
   state = {
     email: "jjj@jjj.com",
     pin: "2437",
```

Comment:
😕 🚩maybe this should be empty strings

Reply:
You're definitely right! We've just been using this so that we can easily log in for testing. I hadn't thought about it much but it probably is bad practice to let it sneak into the PRs .. thanks!